### PR TITLE
FIX: Rename LICENSE to LICENSE.md

### PR DESCRIPTION
**Pull request by brandondombrowsky**
The .md portion didn't merge or I accidently forgot it. It is added now as a quick fix.

### CODE REVIEW

**Review by brinkbrink — CHANGES_REQUESTED** [review]
We can merge this after the previous PR from Brandon is merged.

**Review by brinkbrink — APPROVED** [review]

**Comment by brandondombrowsky** [comment]
This License was already approved and merged. It was simply missing the .md to the file name. The only repos I am having problems on are the belindas-closet-android and nsc-events-android repos.

**Pull request merged**
